frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -154,8 +154,10 @@ editor-UnsavedChanges--leave-anyway = Leave anyway
 
 
 ## Entity Details Navigation
-## Shows next/previous buttons.
+## Copies string link - Shows next/previous buttons.
```

Comment:
Nit: Shows Copy Link and Next/Previous buttons.

Please also update the docstring of the class in EntityDetails.js.

Reply:
Can you please clarify which docstring in EntityDetails.js to update as I did not locate one for the Next/Previous or navigation in general. ~ Thanks

Reply:
Sorry, I meant `EntityNavigation.js`:
https://github.com/mozilla/pontoon/blob/00f69cf509e0bb3b18d9c36c66d13798cac5c930/frontend/src/modules/entitydetails/components/EntityNavigation.js#L19